Page token coingecko_id update

rbpollock opened the pull request:
- Added coingecko_id to ethereum, gravitybridge, and osmosis assetlists
- Removed ".grv" from symbol on osmosis (was being duplicated in frontends)

JeremyParish69 commented:
Please only add the cgid to the original asset listed under Ethereum, but not any others

rbpollock commented:
> Please only add the cgid to the original asset listed under Ethereum, but not any others

I've removed the other commits and added the symbol update for osmosis
